config/expressive.config.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
It uses different config keys.  service_manager is replaced by "dependencies", BUT that doesn't mean we can't include the other config file AND re-key it.  That might be a better way to go here.